src/utils/dateRange.ts

```diff
@@ -56,7 +56,7 @@ export function getForDateRangeString(
 
   return intlHelper(
     intl.formatMessage(message, {
-      groupByValue: intl.formatMessage(messages.GroupByValues, { value, count: 2 }),
+      groupByValue: intl.formatMessage(messages.GroupByValuesTitleCase, { value, count: 2 }),
```

Comment:
This isn't always a group by value. For the details pages, this is a cost.
I believe this is why the key was previously named `value`